packages/react-aria-components/src/index.ts

```diff
@@ -16,7 +16,7 @@ import 'client-only';
 
 export {CheckboxContext, ColorAreaContext, ColorFieldContext, ColorSliderContext, ColorWheelContext, HeadingContext} from './RSPContexts';
 
-export {Autocomplete, AutocompleteContext, AutocompleteStateContext, InternalAutocompleteContext} from './Autocomplete';
+export {UNSTABLE_Autocomplete, UNSTABLE_AutocompleteContext, UNSTABLE_AutocompleteStateContext, UNSTABLE_InternalAutocompleteContext} from './Autocomplete';
```

Comment:
If it's named **Internal**AutocompleteContext, is it meant to be exported? I think eventually we'd want to export it but probably with a different name in that case.

Reply:
I'll remove it from the exports from now and add it to the followup ticket. Perhaps a name like `AutocompleteCollectionContext` or something might appropriate